packages/block-library/src/navigation/edit/use-create-navigation-menu.js

```diff
@@ -57,8 +57,12 @@ export default function useCreateNavigationMenu( clientId ) {
 					);
 				} );
 			}
+
+			const maybeSlug = slug || title;
```

Comment:
Let's explain _why_ this is here. In the future we may want to pass a slug to represent the template part hierarchy.